pkg/devserver/lifecycle.go

```diff
@@ -45,3 +53,60 @@ func (l lifecycle) OnFunctionScheduled(
 		Cron:         event.Cron,
 	})
 }
+
+func (l lifecycle) OnFunctionFinished(
+	ctx context.Context,
+	id state.Identifier,
+	item queue.Item,
+	resp state.DriverResponse,
+	s state.State,
+) {
+	now := time.Now()
+	data := map[string]interface{}{
```

Comment:
FYI, and we do _not_ need to change it, Go 1.20 adds `any` as a nicer keyword for `interface{}`:  `map[string]any`.